packages/storybook/stories/QuickStarts/snippets/QuickstartCompositeAdapter.snippet.tsx

```diff
@@ -38,15 +39,15 @@ function App(): JSX.Element {
         setChatAdapter(
           await createAzureCommunicationChatAdapter({
             endpoint: endpointUrl,
-            userId: { communicationUserId: userId },
+            userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
```

Comment:
I see those down-casts a lot. What happens if one passes a `MicrosoftTeamsUserIdentifier`, will we throw somewhere internally?

Reply:
Where we will throw isn't very well defined. We'll throw when we try to access `communicationUserId` field.

That said, we should really accept the general enum as input as well. When we start supporting `MicrosoftTeamsUserIdentifier`, we will have to do that. But that's a backwards compatible change, so I decided to not change the input signature until we have a real need.